lib/resourcemerge/core.go

```diff
@@ -445,6 +446,13 @@ func ensureSELinuxOptions(modified *bool, existing *corev1.SELinuxOptions, requi
 	setStringIfSet(modified, &existing.Level, required.Level)
 }
 
+func ensureResourceRequirements(modified *bool, existing *corev1.ResourceRequirements, required corev1.ResourceRequirements) {
```

Comment:
include both https://godoc.org/k8s.io/api/core/v1#ResourceRequirements `Limits` and `Requests`

Reply:
@abhinavdahiya ok will update.